Add and rename some data actions (#233)
ServiceNow/azimuth: docs/docs/key-concepts/proposed-actions.md

Five proposed actions are currently supported:

* **Relabel**: The label for this utterance should be changed.
* **Augment with Similar**: This utterance lacks nearby utterances in the same class, and new,
  similarly labeled utterances should be added.

> * **Add New Class**: This data point may belong to a new class that doesn't exist in the

Author: ```suggestion
* **Define New Class**: This data point may belong to a new class that doesn't exist in the
```

> * **Add New Class**: This data point may belong to a new class that is not available in the

Reviewer: I know it's not new with this PR, but I realize that "not available" might be unclear. Like, is it just a question of enabling something? So maybe something like that?
```suggestion
* **Add New Class**: This data point may belong to a new class that doesn't exist in the
```

>   currently defined classes. Based on the number of data points identified with this action, a user

Reviewer: That nice wording makes me wonder if the data action would be better called `define_new_class`, since `add` might sound like adding a new class to the utterance, whereas `define` conveys that it is currently not defined. While we're here... What do you think?

Reviewer 2: FWIW, I like "define new class," both because it is clear and because it avoids all the issues we've discussed about what "add" implies and whether it's clear it's just an annotation!

Author: Totally, define sounds better - thanks!
